test/hybrid2d.jl

```diff
@@ -48,77 +48,83 @@ function hvspace_2D(
     return (hv_center_space, hv_face_space)
 end
 
-@testset "1D SE, 1D FV Extruded Domain ∇ ODE Solve vertical" begin
+# V_face = Covariant3Vector
+# V_center = Covariant12Vector
+# V = V_center + V_face
 
+# divergence(V) = horz_div(V) + vert_div(V)
+# divergence(V) = horz_div(V_center) + horz_div(V_face) + vert_div(V_center) + vert_div(V_face)
+
+
+# 1) horz_div(V_center): project to Contravariant1 + Contravariant2, take spectral derivative
+# 2) horz_div(V_face): project to Contravariant1 + Contravariant2, interpolate to center, take spectral derivative
+#   - will be zero if orthogional geom
+# 3) vert_div(V_center): project to Contravariant3, interpolate to face, take FD deriv
+#   - will be zero if orthogional geom
+# 4) vert_div(V_face): project to Contravariant3, take FD deriv
+
+
+
+
+
+@testset "1D SE, 1D FD Extruded Domain vertical advection operator" begin
+
+    # Advection Operator
+    # c ∂_z f
+    # for this test, we use f(z) = sin(z) and c = 1
+    # => c ∂_z f = cos(z)
     hv_center_space, hv_face_space = hvspace_2D()
-    V =
-        Geometry.Cartesian13Vector.(
-            zeros(Float64, hv_face_space),
-            ones(Float64, hv_face_space),
-        )
 
-    function rhs!(dudt, u, _, t)
+    function rhs!(f)
+        adv = zeros(eltype(f), hv_center_space)
         A = Operators.AdvectionC2C(
-            bottom = Operators.SetValue(sin(-t)),
+            bottom = Operators.SetValue(0.0),
             top = Operators.Extrapolate(),
         )
-        return @. dudt = -A(V, u)
+        return @. adv = A(c, f)
```

Comment:
Although it works, it is better practice to pass `C` as a function argument rather than rely on it being available as a global variable.

Reply:
I agree. I had kept that construction because it was like this in the original version with the ODE

Reply:
I applied the suggested change wherever possible. Some instances were not possible because of the broadcasting issue that was found.